Sources/Storage/Disk/Handler/PersistentHandler.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PersistanceHandler.swift
+//  StatsdClient
+//
+//  Created by Nghia Tran on 10/8/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+
+protocol PersistentHandler {
+
+    var config: DiskConfigurable { get }
+
+    func getTotal() -> Int
```

Comment:
`getTotal` is bad naming, `fileCount` would clearly indicate that we're getting the total number for files